src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserPrefsAccessorConstants_en.properties

```diff
@@ -934,9 +934,12 @@ pythonPathDescription = The path to the default Python interpreter.
 saveRetryTimeoutTitle = Save Retry Timeout
 saveRetryTimeoutDescription = The maximum amount of seconds of retry for save operations.
 
-# Whether the Insert Pipe Operator command should insert the native R pipe operator, |>
-insertNativePipeOperatorTitle = Use R's native pipe operator, |>
-insertNativePipeOperatorDescription = Whether the Insert Pipe Operator command should insert the native R pipe operator, |>
+# Whether the Insert Pipe Operator command should use the native R pipe operator, |>
+useNativePipeOperatorTitle = Use R's native pipe operator, |>
+useNativePipeOperatorDescription = Whether the Insert Pipe Operator command should use the native R pipe operator, |>
+useNativePipeOperatorEnum_never=Never (use magrittr pipe)
+useNativePipeOperatorEnum_only_R41=Only when using R 4.1+
+useNativePipeOperatorEnum_always=Always
```

Comment:
🇫🇷 

```
useNativePipeOperatorTitle = Utiliser le pipe natif, |>
 useNativePipeOperatorDescription = Indique si la commande 'Insérer un opérateur de tube' utilise le pipe natif, |>
 useNativePipeOperatorEnum_never=Jamais (utilise le pipe de magrittr)
 useNativePipeOperatorEnum_only_R41=Seulement avec R 4.1+
 useNativePipeOperatorEnum_always=Toujours
```

Same comment as before, I don't think "tube" is being used

Reply:
But maybe this is out of context for this pr.

Reply:
Thanks! Apparently we can leave the French translations unedited, and leave them for the Stats Canada folks to translate, but since you've already provided the translation I will update the file to use it. Merci!

(Also thanks for the note about `tube` vs. `pipe` -- I'm assuming someone provided the literal translation previously but I will trust your experience of what is more commonly used